AUTHORS

```diff
@@ -2,5 +2,4 @@ Benoît Bryon <benoit@marmelune.net>
 Natim <hubscher.remy@gmail.com>
```

Comment:
You have two entries. One with `Natim` and the other with your fullname. Might as well remove one.
What do you think?

```suggestion
```